CMakeLists.txt

```diff
@@ -2,7 +2,8 @@ cmake_minimum_required(VERSION 2.8)
 project(whycon)
 
 # set some flags
-add_definitions(-std=c++11)
+#add_definitions(-std=c++11)
+add_definitions(-std=gnu++0x)
```

Comment:
any particular reason for this? c++11 is supposed to be more standard. Moreover, I read gnu++0x flag is deprecated in favor of c++11.